specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,711 @@
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id for the resource.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource definition.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the type of the resource definition.",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the resource location."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the properties. Define quantum space's specific properties.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/WorkspaceResourceProperties"
+            }
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "description": "Gets or sets the tags."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "WorkspaceResourceProperties": {
+      "description": "Properties of a Workspace",
+      "properties": {
+        "providers": {
+          "type": "array",
+          "description": "Providers selected for a Workspace",
+          "items": {
+            "$ref": "#/definitions/Provider"
+          }
+        },
+        "usable": {
+          "type": "string",
+          "enum": [
+            "Yes",
+            "No",
+            "Partial"
+          ],
+          "x-ms-enum": {
+            "name": "UsableStatus",
+            "modelAsString": true
+          },
+          "description": "Whether the current workspace is usable."
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "ProviderLaunching",
+            "ProviderUpdating",
+            "ProviderDeleting",
+            "ProviderProvisioning",
+            "Deleted",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningStatus",
+            "modelAsString": true
+          },
+          "description": "Provisioning status field"
+        },
+        "storageAccount": {
+          "type": "string",
+          "description": "ARM Resource Id of the storage account associated with this workspace."
+        }
+      }
+    },
+    "ProviderDescription": {
+      "type": "object",
+      "description": "Information about a Provider.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique Provider's id."
+        },
+        "name": {
+          "type": "string",
+          "description": "Provider's display name."
+        },
+        "properties": {
+          "type": "object",
+          "description": "A list of Provider-specific properties.",
```

Comment:
should just be `"schema": { "$ref" :"#/definitions/ProviderProperties" }` -- use of allOf is messy